### PR TITLE
Pluralize biometrics widget name

### PR DESCRIPTION
**Pull request by denniskigen**
Exactly that.

### CODE REVIEW

File: src/view-components/core-views.tsx [hunk 1]
```diff
@@ -279,7 +279,7 @@ export const coreTabbedViewDefinitions = [
         view: "VitalsSummary"
       },
       {
-        label: "Biometrics",
+        label: "Biometricss",
```
Comment (HerbertYiga) [inline src/view-components/core-views.tsx:282]: do we need a double s around this?

Comment (denniskigen) [reply src/view-components/core-views.tsx]: Nope. Fixed

**Comment by denniskigen** [comment]
Ping @FlorianRappl

**Review by FlorianRappl — APPROVED** [review]
LGTM :beers: :)

**Pull request merged**